### ISSUE TITLE
[BUG] Animals Redone not working

### ISSUE THREAD

**Issue by joao-b-rosario** [dorianpb/cem#99]
0.7.1

- [x] I am using the latest version of CEM

Animals Redone Textures are missing.

I have tried to fix it by removing the Path specification inside .JEM files, but it didn't work.

Expected behavior

![image](https://user-images.githubusercontent.com/43723879/205327082-b5ad8c46-3e77-417d-be05-e095336e3e61.png)

Outcome

![image](https://user-images.githubusercontent.com/43723879/205327439-6fd4ff7f-bee3-42d1-8e0d-45c912e1ef35.png)

 - OS: Windows 11
 - Java Version: 17.0.3
 - Minecraft Version: 1.19.2

**Comment by broadcast** [comment]
I have the same issue on 1.19.2 as well. @Deadrosas were you ever able to resolve this? I'm using fabric loader 0.14.12, fabric api 0.72.0 and latest CEM version. I have a feeling another mod on my list might be breaking this as I have a separate modpack in which CEM is working just fine. Would just mind sharing your mod list so I can cross reference with mine?

**Comment by joao-b-rosario** [comment]
> I have the same issue on 1.19.2 as well. @Deadrosas were you ever able to resolve this? I'm using fabric loader 0.14.12, fabric api 0.72.0 and latest CEM version. I have a feeling another mod on my list might be breaking this as I have a separate modpack in which CEM is working just fine. Would just mind sharing your mod list so I can cross reference with mine?

Yeah i was able to get it working. I can share the files later today.

**Comment by broadcast** [comment]
Ok, I've isolated my issue to the Charm mod being the cause https://www.curseforge.com/minecraft/mc-mods/charm . The problem is that Charm introduces its own mob variant feature for almost every vanilla mob which will conflict with cem + resource pack. Turning off the variant mob settings in Charm fixes the problem for me.
